packages/cozy-konnector-libs/src/helpers/cozy-client-js-stub.js

```diff
@@ -13,20 +13,65 @@ if (fs.existsSync(FIXTURE_PATH)) {
   fixture = require(FIXTURE_PATH)
 }
 
+let DUMP_PATH = 'importedData.json'
+const KONNECTOR_DEV_CONFIG_PATH = path.resolve('konnector-dev-config.json')
+if (fs.existsSync(KONNECTOR_DEV_CONFIG_PATH)) {
+  const KONNECTOR_DEV_CONFIG = require(KONNECTOR_DEV_CONFIG_PATH)
+  DUMP_PATH = path.join(KONNECTOR_DEV_CONFIG.fields.folderPath || '.', DUMP_PATH)
+}
+// Truncate dump file
+fs.writeFileSync(DUMP_PATH, '', 'utf8')
```

Comment:
@Phyks Sorry for the long delay, please just replace : 

```javascript
fs.writeFileSync(DUMP_PATH, '', 'utf8')
```

with : 

```javascript
fs.writeFileSync(DUMP_PATH, '[]', 'utf8')
```

Or else you get a `JSON.parse` error.